yarn.lock

```diff
@@ -11280,7 +11352,7 @@ write@1.0.3:
   dependencies:
     mkdirp "^0.5.1"
 
-ws@^6.2.1:
+ws@^6.0.0, ws@^6.2.1:
```

Comment:
This too?

Reply:
Similar to https://github.com/artsy/horizon/pull/160/files#r457366183.